content/20.results.md

```diff
@@ -91,17 +91,14 @@ Separating honoree results by honor category did not reveal any clear difference
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/f7b8a119faf269f072886bd4e50d533a0913252a/figs/us_racial_makeup.png){#fig:us_racial_makeup}
 
 
-### Reception & Feedback
+### Improvments to Honoree Diversity Subsequent to Our Primary Analysis
 
+While our study was primarily designed to assess the diversity of honorees, the findings raise important questions about what can be done to address the situation.
+The widespread adoption of preprints in biology and medicine enables us to begin to address these questions by examining any changes subsequent to our initial report.
```

Comment:
I'm trying to avoid using "address" too many times.

```suggestion
By publishing our original findings on the biorxiv preprint server, we could begin to answer these questions by examining changes subsequent to our initial report.
```